lib/src/util/map_util.dart

```diff
@@ -68,6 +68,58 @@ Map getPropsToForward(Map props, {
   return propsToForward;
 }
 
+/// Adds unconsumed props to a passed in [Map] reference ([propsToUpdate]).
+///
+/// Based upon configuration, the function will overlook [props] that are not
+/// meant to be passed on, such as non-DOM props or specified values.
+void forwardUnconsumedProps(Map props, {
+  bool omitReactProps: true,
+  bool onlyCopyDomProps: false,
+  Iterable keysToOmit,
+  Iterable<Iterable> keySetsToOmit,
+  Map propsToUpdate,
+}) {
+  if (onlyCopyDomProps) {
+    for (String key in props.keys) {
+      if (key.startsWith('aria-') ||
+          key.startsWith('data-') ||
+          _validDomProps.contains(key)) {
+        propsToUpdate[key] = props[key];
+      }
+    }
+    return;
+  }
+
+  for (String key in props.keys) {
+    if (keysToOmit != null && keysToOmit.contains(key)) continue;
+
+    if (keySetsToOmit != null) {
+        /// If the passed in value of [keySetsToOmit] comes from
```

Comment:
#nit everything inside this `if keySetsToOmit != null` block is indented 4 spaces, should be 2.